addons/website_sale_comparison/views/website_sale_comparison_template.xml

```diff
@@ -195,17 +195,13 @@
                                 </tr>
                                 <tr>
                                     <td t-if="len(attrib_categories)" class='td-top-left border-top-0'/>
-                                    <td t-foreach="products" t-as="product" class="border-top-0">
+                                    <td t-foreach="products" t-as="product" class="border-top-0" style="vertical-align: top;">
```

Comment:
we try to avoid such hardcoded style directives (as they'll be better handled with bootstrap classes/custom css most of the time).

Can't you use `align-top` here instead ? :thinking:

Reply:
Oui! I discovered that on a task I took after but did not remember about this here. Working fine with align-top. Thanks!